cumulus/client/bootnodes/src/schema/response.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+package response;
+
+message Response {
+    // Peer ID of the node on the parachain side.
+    required bytes peer_id = 1;
+
+    // Multiaddresses of the parachain side of the node. The list and format are the same as for
+	// the `listenAddrs` field of the `identify` protocol.
+    repeated bytes addrs = 2;
```

Comment:
nit: Should the addr and peer_id be untrusted as well? (similar to genesis and fork)

Reply:
What do you mean by untrusted here?

Reply:
I believe you placed the `Untrusted` comment for the ones below